alfalfa_worker/Dockerfile

```diff
@@ -21,7 +25,13 @@ RUN apt-get update -y && apt-get install -y \
         gdebi-core \
         python3 \
```

Comment:
Can we simlink python3 to python (and pip)?  Then we can remove the `python3` calls and just say `python`

```
ln -sf /usr/bin/python3 /usr/bin/python
ln -sf /usr/bin/pip3 /usr/bin/pip
```

Reply:
the latter, symlink after install.